src/main/resources/db/changelog/changelog.sql

```diff
@@ -180,3 +180,22 @@ CHANGE COLUMN `passwordAlgorithm` `passwordAlgorithm` ENUM('STORMPATH_HMAC_SHA_2
 ALTER TABLE `AccountSecrets`
 CHANGE COLUMN `algorithm` `algorithm` ENUM('STORMPATH_HMAC_SHA_256', 'BCRYPT', 'PBKDF2_HMAC_SHA_256',
   'STORMPATH_PBKDF2_DOUBLE_HASH') NOT NULL;
+  
+-- changeset bridge:4
+
+CREATE TABLE IF NOT EXISTS `RequestInfos` (
+  `userId` varchar(255) NOT NULL,
+  `clientInfo` varchar(255),
+  `userAgent` varchar(255),
+  `languages` varchar(255),
+  `userDataGroups` varchar(255),
+  `userSubstudyIds` varchar(255),
+  `activitiesAccessedOn` varchar(255),
+  `signedInOn` varchar(255),
+  `uploadedOn` varchar(255),
+  `timeZone` varchar(255),
+  `studyIdentifier` varchar(255) NOT NULL,
+  PRIMARY KEY (`userId`),
+  CONSTRAINT `RequestInfo-UserId-Constraint` FOREIGN KEY (`userId`) REFERENCES `Accounts` (`id`) ON DELETE CASCADE
+) CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+
```

Comment:
Merged changes haven't been added to the pull request yet.

Reply:
Merged